src/wifipaf/tests/TestWiFiPAFLayer.cpp

```diff
@@ -71,6 +71,7 @@ class TestWiFiPAFLayer : public WiFiPAFLayer, private WiFiPAFLayerDelegate, publ
     CHIP_ERROR WiFiPAFMessageReceived(WiFiPAFSession & RxInfo, System::PacketBufferHandle && msg) override { return CHIP_NO_ERROR; }
     CHIP_ERROR WiFiPAFMessageSend(WiFiPAFSession & TxInfo, System::PacketBufferHandle && msg) override { return CHIP_NO_ERROR; }
     CHIP_ERROR WiFiPAFCloseSession(WiFiPAFSession & SessionInfo) override { return CHIP_NO_ERROR; }
+    bool WiFiPAFResourceAvailable(void) override { return true; }
     static constexpr size_t kTestPacketLength     = 100;
```

Comment:
should there be more unit tests? Tons of code changed, the only unit test change is to return a true in a new virtual methods. 

Can't this be unit tested? Summary should explain

Reply:
I will add the case to test it